test/e2e/network/service.go

```diff
@@ -1080,12 +1130,7 @@ var _ = SIGDescribe("Services", func() {
 		podNames2, svc2IP, err := StartServeHostnameService(cs, getServeHostnameService(svc2), ns, numPods)
 		framework.ExpectNoError(err, "failed to create replication controller with service: %s in the namespace: %s", svc2, ns)
 
-		hosts, err := e2essh.NodeSSHHosts(cs)
-		framework.ExpectNoError(err, "failed to find external/internal IPs for every node")
-		if len(hosts) == 0 {
-			framework.Failf("No ssh-able nodes")
-		}
-		host := hosts[0]
+		host := getSSHHost(cs)
```

Comment:
Move `checkHostSupportSSH` here would change the current situation that this case can only run in providers gce/gke/aws/local.

Reply:
I will remove it later. This case will not exist any host var.